src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -22,7 +22,7 @@
         "symfony/http-foundation": "~2.8|~3.0|~4.0",
         "symfony/http-kernel": "^3.3|~4.0",
         "symfony/polyfill-ctype": "~1.8",
-        "twig/twig": "~1.34|~2.4"
+        "twig/twig": "~1.34|~2.9"
```

Comment:
Ah I need to figure out to which version I should bump 1.x

Reply:
Not sure there is such a version… is the plan to backport `apply` on 1.x?

Reply:
nope, you can let it untouched

Reply:
hmm... right, we do need to drop the 1.x compat, not sure it's fine in a patch release. @fabpot?

Reply:
The `apply` tag will be part of Twig 1.40.